LICENSE

```diff
@@ -0,0 +1,21 @@
+The MIT License (MIT)
```

Comment:
I think we should use Apache since the original was Apache.

Reply:
Noticed now that the original project also uses MIT, only uses the Apache license for the Makefile, which was probably copied from another project.

I kept the Apache license on the same file and added a modified notice.